lasagne/layers/conv.py

```diff
@@ -15,6 +15,8 @@
     "TransposedConv2DLayer",
     "Deconv2DLayer",
     "DilatedConv2DLayer",
+    "TransposedConv3DLayer",
+    "Deconv3DLayer"
```

Comment:
Don't forget the comma in the end, so we can add more lines without touching the existing ones.